javascript/src/index.ts

```diff
@@ -38,11 +41,22 @@ import * as base64 from "@stablelib/base64";
 import * as sha256 from "fast-sha256";
 
 const WEBHOOK_TOLERANCE_IN_SECONDS = 5 * 60; // 5 minutes
+const VERSION = "0.25.0";
```

Comment:
We don't use a packager so this I don't know of another option here but in the future we could use rollup/webpack and inject this via a plugin